webpack.mix.js

```diff
@@ -1,7 +1,7 @@
 const mix = require('laravel-mix');
 
 // Public path helper
-const public = path => `${mix.config.publicPath}/${path}`;
+const publicPath = path => `${mix.config.publicPath}/${path}`;
 
 // Source path helper
 const src = path => `resources/assets/${path}`;
```

Comment:
Maybe `assetPath` to keep things uniform? And then we could perhaps just put them together under `// Paths`

Reply:
I was initially trying to keep the function names short so it would be easier than writing out `resources/assets`. I’m game with changing it though.